client/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:10-alpine
+
+RUN mkdir /usr/src/client -p
+
+WORKDIR /usr/src/client
+
+RUN npm install -g yarn
+
+RUN apk add --update python
+
+RUN apk add --update make
+
+RUN apk add --update g++
+
+RUN apk add --update openjdk11-jre
```

Comment:
You could && the run commands in order to create fewer layers. But this may be done alongside issue #83

Reply:
Done.